### ISSUE TITLE
@version in docblock

### ISSUE THREAD

**Issue by neooblaster** [phpDocumentor/phpDocumentor2#1844]
Hi phpDoc Team,

Sorry to raised an issue to ask a little question.
I'm trying to make a template file with DocBlock presentation for my PHP files.
In my DocBlock i want to use @version tag, but i would like to put it behind Summary.

Below : What i would like to do :
```php
/**
 * Last Update : xx.xx.2017.
 *
 * @version 1.0.0 Description_here
 *
 * ## Version 1.0.0 :: xx.xx.2017
 * 
 * • A
 *
 * • B
 * 
 * 
 * ## Description Fonctionnelle
 *
 * 
 * ## Détail de la fonction
 * 
 * @param void 
 * 
 * @return void
 **/
```

Below what i get :

![image](https://cloud.githubusercontent.com/assets/22078558/26019729/fb9fa2d4-3777-11e7-964f-a005a64635ff.png)

I would like to have Version code, but have advanced details like this :

```php
/**
 * Last Update : xx.xx.2017.
 *
 * ## Version 1.0.0 :: xx.xx.2017
 * 
 * • A
 *
 * • B
 * 
 * 
 * ## Description Fonctionnelle
 *
 * 
 * ## Détail de la fonction
 * 
 * @param void 
 * 
 * @return void
 **/
```

![image](https://cloud.githubusercontent.com/assets/22078558/26019748/295bc978-3778-11e7-9f78-3466b3450da9.png)

Is there a way to send breaking signal for @version tag or the only way to use it, is at the end of DocBlock or to use a new tag ?

I'm on phpdoc v2.8.5

Another question is about keyword "File"
Is possible to display filename where function is located ?

![image](https://cloud.githubusercontent.com/assets/22078558/26020637/4e0bbc84-3781-11e7-9534-55cf0c0cc4e4.png)

Thanks for your support.
Best regards
Nicolas DUPRE

**Comment by mvriel** [comment]
Hello Nicolas,

If I understand your question correctly then you would like the Summary, the Version tag, then the Description and finally the other tags; am I right about this?

If so, I am afraid this is not supported in the DocBlock standard. DocBlocks are divided into three sections: Summary, Description, and Tags. Having a tag before the description is not possible because the text accompanying a tag (its own description) may be multi-line and the delimiter to know where that text ends is the next tag.

So in short: I am afraid this is not possible without changing the standard.

With regards to the missing Filename; this seems to be a bug in the Clean template. Can you open a separate issue for this so that we can fix it? Thanks in advance!

**Comment by neooblaster** [comment]
Hi mvriel,

Thanks for your reply.
You have well understood my question.
I'm beginner with phpDocument & DocBlock, that why i ask that newbie question.
I perfectly understand your answer and how that works behind.
That could be annoying to write ending tag for each block ;)

For filename on Clean Template, i didn't though that was a bug. I thought i had missing data. So i will open issue for this.

Thanks a lot for your reply.
